test/dump/const.txt

```diff
@@ -242,15 +242,15 @@ Code Disassembly:
  00002f: 1a                         | drop
  000030: 42 00                      | i64.const 0
  000032: 1a                         | drop
- 000033: 42 80 80 80 80 80 80 80 80 | i64.const -9223372036854775808
+ 000033: 42 80 80 80 80 80 80 80 80 | i64.const 9223372036854775808
  00003c: 80 7f                      | 
  00003e: 1a                         | drop
- 00003f: 42 7f                      | i64.const -1
+ 00003f: 42 7f                      | i64.const 18446744073709551615
  000041: 1a                         | drop
- 000042: 42 80 80 80 80 80 80 80 80 | i64.const -9223372036854775808
+ 000042: 42 80 80 80 80 80 80 80 80 | i64.const 9223372036854775808
  00004b: 80 7f                      | 
  00004d: 1a                         | drop
- 00004e: 42 7f                      | i64.const -1
+ 00004e: 42 7f                      | i64.const 18446744073709551615
```

Comment:
Are you sure it makes more sense to show these as unsigned?  

The numbers are read signed values in the binary reader so maybe it makes sense to show them as signed in the disassembly too?

https://github.com/WebAssembly/wabt/blob/2428315bab6d71b4ec794f5519374f2448cabb44/src/binary-reader.cc#L827